### ISSUE TITLE
Bonds should be drawn with draw_cylinders

### ISSUE THREAD

**Issue by adrianusler** [Molara-Lab/Molara#313]
Currently, the drawing of bonds happens somewhere else, apparently. Even when draw_cylinders is completely deactivated, bonds still work...

**Comment by Michel-Heinz** [comment]
Yes this is true! However, I did this to write a separate routine to speed up the drawing of large or a lot of molecules. I would not like to combine these things. Draw_spheres and draw_cylinders were programmed to draw small objects not computationally expensive ones.

**Comment by adrianusler** [comment]
Couldn't we wrap your more performant code into those routines?

**Comment by Michel-Heinz** [comment]
I cannot remember what I wrote there, only that it was not that easy to merge these bits.
